### PR TITLE
[CMS-614] Update secrets file name and docker images.

### PR DESCRIPTION
**Pull request by kporras07**
This seems to be the right filename for pushback: https://github.com/pantheon-systems/quicksilver-pushback/blob/main/push-back.php#L22

I also updated docker images to use 8.x

### CODE REVIEW

**Comment by kporras07** [comment]
It fails with new filename.

Looks like there is a bug in secrets plugin when trying to set secrets in a file not directly in files/private: https://github.com/pantheon-systems/terminus-secrets-plugin/blob/1.x/src/Commands/SecretsCommand.php#L207 (file_put_contents fails because of missing folder)

See https://github.com/kporras07/cms-578-bt3/runs/5518686494?check_suite_focus=true#step:10:74

**Comment by kporras07** [comment]
Fixed the above error. Also, added a conditional to use the right github token if github actions.

File: d9/.ci/deploy/pantheon/dev-multidev [hunk 1]
```diff
@@ -38,8 +38,12 @@ terminus -n drush "$TERMINUS_SITE.$TERMINUS_ENV" -- cr
 terminus -n env:clear-cache $TERMINUS_SITE.$TERMINUS_ENV
 
 # Ensure secrets are set
-terminus -n secrets:set "$TERMINUS_SITE.$TERMINUS_ENV" token "$GITHUB_TOKEN" --file='github-secrets.json' --clear --skip-if-empty
-
+if [[ ! -z "$GH_TOKEN" ]]
```
Comment (greg-1-anderson) [inline d9/.ci/deploy/pantheon/dev-multidev]: Why do we have both `$GH_TOKEN` and `$GITHUB_TOKEN`?

These blocks could be less verbose:
```
terminus -n secrets:set "$TERMINUS_SITE.$TERMINUS_ENV" token "${GH_TOKEN:-$GITHUB_TOKEN}" --file='.build-secrets/tokens.json' --clear --skip-if-empty
```

Comment (kporras07) [reply d9/.ci/deploy/pantheon/dev-multidev]: GITHUB_TOKEN is the name we normally use with every CI provider; however, it's set by default with Github Actions to a read-only token for the current repo. Then, we set a new GH_TOKEN that should be used as secret in this scenario

**Review by greg-1-anderson — APPROVED** [review]

**Pull request merged**